lib/collection/src/update_handler.rs

```diff
@@ -634,6 +647,20 @@ impl UpdateHandler {
             }
             let ack = confirmed_version.min(max_ack);
 
+            let clock_map_store_future = async {
+                let clock_map_tmp_file = tempfile::NamedTempFile::new()?;
+                let clock_map_tmp_path = clock_map_tmp_file.into_temp_path();
+
+                clock_map.lock().await.store(&clock_map_tmp_path)?;
+                tokio::fs::copy(&clock_map_tmp_path, &clock_map_path).await?;
+
+                CollectionResult::Ok(())
+            };
```

Comment:
I'd prefer to use the `AtomicFile` wrapper here instead, because we also use this in other places.

Reply:
Totally forgot we have it. Will do.

Reply:
Done.